Pocket/pocket-ios
# Add Tipkit Tips for Swipe to Archive and Swipe Highlights features

## Summary
* This PR takes advantage of `TipKit` to:
    - add a system to easily add tips to any `UIViewController` subclass
    - add tips to help discover two features:
        - Swipe to Archive, on Saves
        - Swipe Highlights, in the Reader

## References 
* Branch name

## Implementation Details
* Add two concrete implementation of `Tip`:
     - `SwipeHighlightsTip`
     - `SwipeArchiveTip`
* Add `TippableViewController` a protocol to easily add tips to `UIKit` screens

## Test Steps
* Build/run this branch
* Navigate to Saves and make sure you see the Swipe to Archive tip (see video below)
* Open any article and make sure you see the Swipe Highlights tip (see video below)

## PR Checklist:

## Video
<p align=center>
<img width=200 height=400 src=https://github.com/Pocket/pocket-ios/assets/34376330/ec3530dd-3f3f-43b2-9e22-966e7abfe6c1>
</p>
